### ISSUE TITLE
Tables of p values

### ISSUE THREAD

**Issue by smiths** [liresearch/climate#12]
Are all of the tables of p values necessary?  It is difficult to get much out of reading a table.  Is there any data in the tables that the reader would actually be looking for?  Maybe a simple summary statement about the highest p value would be enough?

I haven't yet read the part of the paper referencing these tables. I just thought I would give you my high level comment.  Please feel free to ignore it.  😄

**Comment by liresearch** [comment]
@luongcn maybe we can use an monthly average (instead of 12 value for the 12 months) for each temperature variable and each station? (Although the averaged p-value may not be statistically meaningful. )

**Comment by smiths** [comment]
The reason I mentioned the tables is because it is difficult for me to tell what story they are telling.  A quick scan of the paper shows that the tables are only referenced in the following text:

"Hypothesis tests were also used to validate the data in twelve stations. The p-values calculated were very low (Table 4, Table 5 & Table 6)."

If the only point is to get across that the p values are low, this could be done by listing the maximum p-value, without any tables.  The most important thing seems to be that the p-values are all below 0.05.  However, I might be oversimplifying.  Maybe the reader is supposed to extract something else from the tables?  Maybe there is some kind of relevant trend?  If that is the case, some text should likely be added to point out to the reader what it is they should be "seeing" in the data.

**Comment by luongcn** [comment]
You are right @smiths , what we want to tell here is the p-values are very low, so that the hypothesis test is not significant. Therefore, we use confident interval to estimate the error of the multi-model simulation. I will update the maximum p-value and state that most of the p-values are below 0.05 (significant level). 
Should we upload the tables of p-values to Mendeley just in case the reviewers require? @liresearch, @smiths

**Comment by liresearch** [comment]
I think lising the maximum p-value is a very good idea. @luongcn at this point, I don't think we need to upload the table of p-values to Mendeley.

**Comment by luongcn** [comment]
I have changed the text of p-values results. Me and Aly will finish the draft up and upload to GitHub

**Comment by smiths** [comment]
I'm sorry to re-open this issue, but I just looked at the values in the table, instead of only reading the text in the paper.  The text says, "Since the p-values are less than 0.05, the null hypothesis is likely to be true."  I read this as saying that all of the p-values are less than 0.05.  This is not the case.  For Table 4 about 30% of the p-values are greater than 0.05.  For Table 5 almost 25% of the values are greater than 0.05, and for Table 6, about 15% are above 0.05.  I think we need to be careful in our wording, so that other readers do not come to the same incorrect conclusion that I did.  

I think I might be changing my mind on Tables 4, 5 and 6.  The statistical story is not as simple as I thought it was.  The reader might want to judge for themselves how much faith they put in the stats.  In the past where I have displayed tables of p-values, I have highlighted those that exceeded my test threshold.  This lets the reader judge for themselves.  

When I suggested stating the maximum p-value, I was assuming (based on my reading of the text) that the maximum was less than 0.05.  The maximums are actually much larger than this.  I see some entries as high as 0.945.  If this value is given out of context, most people are not going to be much faith in the stats.

If we do leave out the tables, then we will need a longer summary than the p-values are less than 0.05.  We could maybe do the reverse of what I did above and state the percentage of p-values that are below 0.05?

**Comment by luongcn** [comment]
Me and Aly have considered the p-values are higher than 0.05. In our summary, we gave the range of the p-values. Also, we stated that although some of the p-values are very high, most of them (90 out of 144 in tas, 103 out of 144 in tasmax, 122 out of 144 in tasmin) are below 0.05. Therefore, the hypothesis test is not sufficient. That also relates to my question before that whether we should post the tables on Mendaley
